pom.xml

```diff
@@ -336,6 +345,11 @@
                     <artifactId>wildfly-feature-pack-build-maven-plugin</artifactId>
```

Comment:
Do we still need the `wildfly-feature-pack-build-maven-plugin` plugin and some of the other WF provisioning plugins?

Reply:
No, I thought I removed those. Let me re-check.